wc3-hw1-TranNgocLam/cypress/pages/registrationForm.js

```diff
@@ -0,0 +1,279 @@
+                case "Sports":
+                    cy.get(this.CHK_HOBBIES).check("1", {force: true} )
+                    break
+                case "Reading":
+                    cy.get(this.CHK_HOBBIES).check("2", {force: true} )
+                    break
+                case "Music":
+                    cy.get(this.CHK_HOBBIES).check("3", {force: true} )
+                    break
+            }
+        }
+        return this;
+    },
+
+    uploadPicture(picture) {
+        cy.get(this.DLG_UPLOADPICTURE).selectFile(picture)
+        return this
+    },
+
+    inputCurrentAddress(currentAddress) {
+        cy.get(this.TXT_CURRENTADDRESS).type(currentAddress, { force: true });
+        return this;
+    },
+
+    inputState(state) {
+        cy.get(this.DDL_STATE).type(state, { force: true }).type('{enter}', { force: true })
+        return this;
+    },
+    inputCity(city) {
+        cy.get(this.DDL_CITY).type(city, { force: true }).type('{enter}', { force: true })
+        return this;
+    },
+
+    clickSubmit() {
+        cy.get(this.BTN_SUBMIT).click({ force: true });
+        return this;
+    },
+    checkFirstNameValid(valid) {
+        if (valid){
+
+            cy.get(this.TXT_FIRSTNAME).should('have.css', 'border-color', this.GREEN);
+        }else{
+
+            cy.get(this.TXT_FIRSTNAME).should('have.css', 'border-color', this.RED);
+        }
+        return this
+    },
+
+    checkLastNameValid(valid) {
+        if (valid){
+
+            cy.get(this.TXT_LASTNAME).should('have.css', 'border-color', this.GREEN);
+        }else{
+
+            cy.get(this.TXT_LASTNAME).should('have.css', 'border-color', this.RED);
+        }
+        
+        return this
+    },
+
+    checkEmailValid(valid) {
+        if (valid){
+
+            cy.get(this.TXT_EMAIL).should('have.css', 'border-color', this.GREEN);
+        }else{
+
+            cy.get(this.TXT_EMAIL).should('have.css', 'border-color', this.RED);
+        }
+        return this
+    },
+    checkMobileValid(valid) {
+        if (valid){
+
+            cy.get(this.TXT_USERNUMBER).should('have.css', 'border-color', this.GREEN);
+        }else{
+
+            cy.get(this.TXT_USERNUMBER).should('have.css', 'border-color', this.RED);
+        }
+        return this
+    },
+    isGenderChecked(valid) {
+        if (valid){
+
+            cy.get(this.LBL_GENDER).should('have.css', 'border-color', this.GREEN);
+        }else{
+
+            cy.get(this.LBL_GENDER).should('have.css', 'border-color', this.RED);
+        }
+        return this
+    },
+}
+
+export const confirmForm = {
+    BTN_CLOSE: "#closeLargeModal",
+    LBL_STUDENTNAME: "table tbody tr:nth-child(1) td:nth-child(2)",
+
+    isCorrectName(firstName, lastName) {
+        let name = firstName + ' ' + lastName
+        cy.get(this.LBL_STUDENTNAME)
+            .should('have.text', name)
+        return this
+    },
+
+    isCorrectEmail(email) {
+        cy.get('table tbody tr')
```

Comment:
You use same locator many times, define above so if locator change you just have to change in one place